lib/common.py

```diff
@@ -260,6 +260,9 @@ def xo_server_reconnect(self):
         xo_cli('server.disable', {'id': self.xo_srv_id})
         xo_cli('server.enable', {'id': self.xo_srv_id})
         wait_for(self.xo_server_connected, timeout_secs=10)
```

Comment:
Is this still necessary with the new code?

Reply:
Which new code? XO?